[ADD] websites: add plausible.io Dashboard documentation (#2007)
odoo/documentation: content/applications/websites/website/optimize/plausible.rst

> =======================================================
> How to track your website's traffic with Plausible.io
> =======================================================

Reviewer: ```suggestion
=====================================================
How to track your website's traffic with Plausible.io
=====================================================
```

To follow your website's traffic with Plausible.io:

> - `Create a Plausible.io account <https://plausible.io/register/>`__ if   you don't have any.

Reviewer: ```suggestion
- `Create a Plausible.io account <https://plausible.io/register/>`__ if you don't have any.
```

> - `Create your website <https://plausible.io/sites/new/>`__ and `Start collecting data`  if it is not already done.

Reviewer: ```suggestion
- `Create your website <https://plausible.io/sites/new/>`__ and `Start collecting data` if it is not already done.
```

> - Go through the `list of yours websites <https://plausible.io/sites/>`__ and click on the gear wheel.

Reviewer 2: ```suggestion
- Go through the `list of your websites <https://plausible.io/sites/>`__ and click on the gear icon.
```

>     .. image:: media/plausible_cog.png
>         :align: center

Reviewer: ```suggestion
    .. image:: media/plausible_cog.png
       :align: center
```

- Click on the button `+ New Link` in the tab Visibility to create your Shared links.

>     .. image:: media/plausible_new_sharedlink.png
>         :align: center

Reviewer: ```suggestion
    .. image:: media/plausible_new_sharedlink.png
       :align: center
```

- Choose a name (and NO password - it is not supported to embed it in Odoo) and click on the button `Create Shared Link`

>     .. image:: media/plausible_create_sharedlink.png
>         :align: center

Reviewer: ```suggestion
    .. image:: media/plausible_create_sharedlink.png
       :align: center
```

- Copy the new Shared Link url from Plausible.io and paste it in the Settings of Odoo

>     .. image:: media/plausible_copy_sharedlink.png
>         :align: center

Reviewer: ```suggestion
    .. image:: media/plausible_copy_sharedlink.png
       :align: center
```

>     .. image:: media/plausible_paste_sharedlink.png
>         :align: center

Reviewer: ```suggestion
    .. image:: media/plausible_paste_sharedlink.png
       :align: center
```

> .. note:: You are on the Odoo Saas Platform ?

Reviewer 2: ```suggestion
.. note:: You are on the Odoo Saas Platform?
```

    When you `create your database on www.odoo.com/trial <https://www.odoo.com/trial>`__, we create it for you on our own server.
    You don't have anything to do.

> .. note:: What's about the Goal ?

Reviewer 2: ```suggestion
.. note:: What about the Goal?
```
(and maybe just `What about Plausible Goal(s)`?)
